### PR TITLE
feat(communications): add no data block component

### PR DESCRIPTION
**Pull request by govindarajkp**
image

### CODE REVIEW

File: react/src/Communications/NoDataBlock/NoDataBlock.js [hunk 1]
```diff
@@ -1,7 +1,69 @@
 import React from 'react'
 
-function NoDataBlock() {
-  return <>NoDataBlock goes here.</>
+function NoDataBlock({text = 'No Results Found' , overwriteWidth='480', overwriteHeight= '300'}) {
+  return <>
+    <div className='items-center w-96'>
+      <svg version="1.1" viewBox="0 0 1340 1288" width={overwriteWidth} height={overwriteHeight} xmlns="http://www.w3.org/2000/svg">
```
Comment (send2alamelu) [inline react/src/Communications/NoDataBlock/NoDataBlock.js:6]: this SVG doesn't look good, can we try from clip art?

File: react-examples/stories/Communications/NoDataBlock.stories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    overwriteWidth: '',
+    overwriteHeight: ''
+  },
+};
+
+const Template = (args) => <NoDataBlock {...args}/>;
+
+export const Default = Template.bind({});
+Default.args = {};
+
```
Comment (send2alamelu) [inline react-examples/stories/Communications/NoDataBlock.stories.js:17]: can we have 2 -3 variations? adding additional TwixtButton as a children

For example:

1. When there is Job Opening List is empty and we use this no data block but we need to add button to refine search like "Clear Filter (secondary), Search Again (Primary)"

**Review by send2alamelu — CHANGES_REQUESTED** [review]

**Comment by govindarajkp** [comment]
image
image

Added Two type of images

**Comment by govindarajkp** [comment]
image

**Pull request closed without merging**